tests/e2e/pageobjects/ide/ProjectTree.ts

```diff
@@ -235,13 +235,16 @@ export class ProjectTree {
 
         for (let i = 0; i < attempts; i++) {
             // do five checks of the item in one fifth of the time given for root folder item (was causing frequent reloads of the workspace)
-            const isProjectFolderVisible = await this.driverHelper.waitVisibilityBoolean(rootItemLocator, 5, visibilityItemPolling / 5);
+            const isRootFolderVisible = await this.driverHelper.waitVisibilityBoolean(rootItemLocator, 5, visibilityItemPolling / 5);
```

Comment:
Did you check it with multi-project workspaces?

Reply:
I'm just renaming a variable here.

Reply:
if you mean the surrounding changes, I can revert them, but this way we have a specific failure based on which element the method failed to locate

Reply:
Okay, let's keep it as it.